frontend/javascripts/oxalis/controller.js

```diff
@@ -113,8 +113,9 @@ class Controller extends React.PureComponent<PropsWithRouter, State> {
       .then(() => this.modelFetchDone())
       .catch(error => {
         this.props.setControllerStatus("failedLoading");
-        // Don't throw errors for errors already handled by the model.
-        if (error !== HANDLED_ERROR) {
+        // Don't throw errors for errors already handled by the model
+        // or when there is no logged in user since then the initialization fails anyways.
+        if (error !== HANDLED_ERROR && this.props.user != null) {
```

Comment:
Hmm, I'm not sure whether this is the best approach. Basically, this will always tell a user to log in, if an error happened while being not logged in. However, there could be an error which is unrelated to authorization.
For example, if I'm not registered and I view a publicly available dataset and that fails for some reason, I'll be asked to log in.  However, in this case we should show the actual error.

I think, a better approach would be to refine this if-statement à la:

```
const isAuthorizationError = ...;
const encourageLogIn = this.props.user == null && isAuthorizationError;
if (error !== HANDLED_ERROR && !encourageLogIn) {

}
```

The only remaining bit would be to find out how to set `isAuthorizationError`. I'd hope something like `error.statusCode` would be useful for this.

Reply:
Let me know in case we should discuss this in person!